Shared.EventStore/EventStore/EventStoreContext.cs

```diff
@@ -341,12 +119,13 @@ public async Task<String> GetStateFromProjection(String projectionName)
         /// <param name="streamName">Name of the stream.</param>
         /// <param name="expectedVersion">The expected version.</param>
         /// <param name="aggregateEvents">The aggregate events.</param>
-        /// <returns></returns>
+        /// <param name="cancellationToken">The cancellation token.</param>
         public async Task InsertEvents(String streamName,
-                                       Int32 expectedVersion,
-                                       List<DomainEvent> aggregateEvents)
+                                       Int64 expectedVersion,
+                                       List<DomainEvent> aggregateEvents,
+                                       CancellationToken cancellationToken)
         {
-            await this.InsertEvents(streamName, expectedVersion, aggregateEvents, null);
+            await this.InsertEvents(streamName, expectedVersion, aggregateEvents, null, cancellationToken);
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/stuart_ferguson1/Shared/pullRequest?prid=5765119)